src/main/frontend/components/repo.cljs

```diff
@@ -173,6 +173,9 @@
               [:p.pt-2.text-sm.opacity-50
                (t :git/version) (str " " version/version)]]])))])))
 
+(defn shorten-repo-name [repo-name]
+  (last (string/split repo-name #"/")))
```

Comment:
Is windows path handled here?

Reply:
Sorry I missed this question!

You're right, I don't think it handles the windows path. I just reused the same line that was already in the app (https://github.com/logseq/logseq/pull/3297/files#diff-c10c0eb252b25dde7fd97de8929a68e7e11e3f52ca0ccd0fbb54bbf0866e0318L253-L254) and extracted it as a function. Happy to add that, but just calling out that it's not adding new behavior / changing the existing behavior in that sense.